### ISSUE TITLE
install pycocoevalcap

### ISSUE THREAD

**Issue by danielabd** [aditya12agd5/convcap#14]
Hi,
from where need to install the package pycocoevalcap which used in convcap/evaluate.py by "from pycocoevalcap.eval import COCOEvalCap"?

I didn't succeed to install it by pip install.
need I to clone a repository contain it like in:
https://github.com/tylin/coco-caption
?

thanks.

**Comment by jyotianeja** [comment]
Yes. If you clone our repo with -recursive flag, it should work fine.
